kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -3749,6 +3758,51 @@ messages:
       return;
    }
 
+    UserMoveInventoryItem(what=$,where=$)
+   "User moved an item in their inventory. Assuming everything checks out, "
+   "we move it in the plPassive list also."
+   {
+      local iWhere, lSafeList;
+
+      if what = $
+         OR where = $
+         OR what = where
+         OR plPassive = $
+      {
+         return;
+      }
+
+      % Get list position for the drop location.
+      iWhere = FindListElem(plPassive,where);
+
+      % MUST reassign this to plPassive, in case we're deleting the first
+      % list node. Use a temp list to verify there were no issues.
+      lSafeList = DelListElem(plPassive,what);
+      if lSafeList <> $
+      {
+         plPassive = lSafeList;
+      }
+      else
+      {
+         Debug("ALERT! Player ",self,Send(self,@GetTrueName)," tried to set ",
+               "plPassive to $ when deleting a list element! ",plPassive,what);
+      }
+
+      % MUST reassign this to plPassive, in case we're creating the first list node.
+      lSafeList = AddListElem(plPassive,iWhere,what);
+      if lSafeList <> $
+      {
+         plPassive = lSafeList;
+      }
+      else
+      {
+         Debug("ALERT! Player ",self,Send(self,@GetTrueName)," tried to set ",
```

Comment:
This also doesn't make sense to me.. how could this trigger?

Reply:
Removed this section of code.  I'm speculating it was out of some sort of security concern.  It was part of the PR I imported.